components/Logos.tsx

```diff
@@ -13,7 +13,7 @@ export interface Props {
 export default function Logos({ logos }: Props) {
   return (
     <>
-      <div className="flex flex-wrap items-center justify-center">
+      <div className="flex flex-wrap items-center justify-center space-y-4">
```

Comment:
using `gap` with `flex` or `grid` is better as that still applies if you change flex direction for mobile/desktop